src/Entity/MyTeam/DelegatedAccess.php

```diff
@@ -70,9 +71,7 @@ class DelegatedAccess
     /**
      * @var array
      *
-     * @ORM\Column(type="simple_array")
-     *
-     * @Assert\Count(min=1)
+     * @ORM\Column(type="simple_array", nullable=true)
```

Comment:
why do we need to have nullable access?

Reply:
this is a request from Victor. We want to be able to create a member of a team without any accesses